### DISCUSSION TITLE
Is the whole-brain electric field in SimNIBS calculated based on voxels or generated through interpolation?

### DISCUSSION THREAD

**Discussion by lingoodluck** [simnibs/simnibs#460] [9. Results and analysis] [answered]

**Comment by TorgeW** [comment, marked as answer]
A voxel segmentation is used to create the tetrahedral mesh using CHARM (the .msh file). This mesh is the used with the FEM to calculate the electric field in the hole head. The electric field can then be interpolated to every point needed. It can also be translated back into voxel space.

**Reply by lingoodluck** [reply]
Thank you for your answer！
